README.md

```diff
@@ -1,8 +1,15 @@
-# Persian Typo Mistake 🇮🇷
-
-[![MIT License](https://img.shields.io/apm/l/atomic-design-ui.svg?)](https://github.com/tterb/atomic-design-ui/blob/master/LICENSEs)
-
-<img src="./screenshot.png" width="420px" height="100%" alt="persian mistake typo screenshot" title="persian mistake typo screenshot"/>
+<h1 align="center">Persian Typo Mistake 🇮🇷</h1>
+
+<p align="center">
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/hadizz/persian-typo-mistake-extension?style=for-the-badge">
```

Comment:
Sorry but i don't like this style :) I prefer flat style

example:
<img alt="GitHub issues" src="https://img.shields.io/github/issues/hadizz/persian-typo-mistake-extension?style=flat">

Reply:
but like I said, great job with README too! I appreciate it.